spanner/tests/system/test_system.py

```diff
@@ -282,6 +283,33 @@ def test_create_database(self):
             for database in Config.INSTANCE.list_databases()]
         self.assertIn(temp_db_id, database_ids)
 
+    def test_table_not_found(self):
+        temp_db_id = 'temp_db' + unique_resource_id('_')
+
+        table_name1 = 'MyTable'
+        table_name2 = 'NotMyTable'
+        self.assertNotEqual(table_name1, table_name2)
+
+        create_table = (
+            'CREATE TABLE {} (\n'
+            '    Id      STRING(36) NOT NULL,\n'
+            '    Field1  STRING(36) NOT NULL\n'
+            ') PRIMARY KEY (Id)')
+        create_table = create_table.format(table_name1)
+        create_index = 'CREATE INDEX IDX ON {} (Field1)'.format(table_name2)
+
+        temp_db = Config.INSTANCE.database(
+            temp_db_id,
+            ddl_statements=[
+                create_table,
+                create_index,
+            ],
+        )
+        with self.assertRaisesRegexp(NotFound,
+                                     '404 Table not found: '
+                                     + table_name2):
```

Comment:
@chemelnucfin Please ditch the `assertRaisesRegexp` and just check the entire `args` of the caught exception (unless there is some strange opaque ID).

(See my previous [comment][1])

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/pull/4557#discussion_r157915661